ui/css/treeherder.css

```diff
@@ -88,8 +88,8 @@ input:focus::-moz-placeholder {
 
 .th-top-nav-options-panel {
     background-color: white;
-    padding: 10px;
-    border: 1px solid black;
+    padding: 10px 10px 0;
+    border-bottom: 1px solid black;
```

Comment:
I removed the 1px surround, for all panels here (Sheriff, Filter) since the top, left, right, aren't really needed. It gives us a little more space. Arguably the bottom isn't needed either, but I left it in, in case the user opens both the Sheriff and Filter panels together (the line will divide the two).